src/oops/SOLID/openClosePrinciple/before/taxes/FullTimeEmpTaxPayable.java

```diff
@@ -0,0 +1,19 @@
+package oops.SOLID.openClosePrinciple.before.taxes;
+
+import oops.SOLID.openClosePrinciple.before.employees.Employee;
+
+public class FullTimeEmpTaxPayable implements TaxPayable{
+
+    private final double INCOME_TAX_PERCENT = 30;
+    private final double PROFESSIONAL_TAX_PERCENT = 2;
+    private final double EDUCATIONAL_CESS_PERCENT = 1;
+
+    @Override
+    public double calculateTax(Employee employee) {
+        int monthlySalary = employee.getMonthlyIncome();
+        return
+                (INCOME_TAX_PERCENT * monthlySalary)/100
+                + (PROFESSIONAL_TAX_PERCENT * monthlySalary)/100
+                + (EDUCATIONAL_CESS_PERCENT *monthlySalary)/100;
```

Comment:
Although not related to this assignment directly, wrong formula though. Cess is applied as a percentage of tax and not the salary. ;)

Reply:
Didn't know that, thanks :)